src/sync/outgoingSyncUtils.js

```diff
@@ -173,6 +173,8 @@ const generateSyncData = (settings, recordType, record) => {
       };
     }
     case 'Transaction': {
+      const isCashReconciliation = record.type === 'payment' || record.type === 'receipt';
```

Comment:
could just do `record.totalPrice || record.total`

Reply:
I went with explicit mainly for readability but also just because I think it's a bit safer in terms of potential regressions or weird edge cases. I agree with above re. removing `totalPrice`, so my inclination is to keep it this way and consider it as just a short term workaround for UAT.